### ISSUE TITLE
python indent: vim overrides user-specified dedentation after parenthetical statement

### ISSUE THREAD

**Issue by bugfood** [vim/vim#6060]
**Describe the bug**
Vim's python indentation rules indent a line following a parenthetical statement to the same level as the beginning line of that statement; this can be incorrect when the following line is intentionally dedented.

**To Reproduce**
1. Create indent.py with contents:
```
class Foo:
    def foo1():
        return (1
                + 2)

    def bar1():
        pass

    def foo2():
        return (
            1,
            2,
        )

    def bar2():
        pass
```
2. `vim indent.py`
3. Position cursor on the line `def bar1():`.
4. Press `==`. Observe that vim indents the line further.
5. Repeat for the line `def bar2():`, which is a variant.

**Expected behavior**
Vim should not further indent either line, since they are already indented properly.

**Environment (please complete the following information):**
 - Vim version 8.2
 - Debian Sid
 - Terminal: rxvt-unicode

**Additional context**
I will provide a patch for this.

Thanks,
Corey

**Comment by bugfood** [comment]
PR to fix this:
https://github.com/vim/vim/pull/6061

Thanks,
Corey

**Comment by dkearns** [comment]
This was fixed via the referenced PR and 47e13953ffdbb9f163b901196dec8c2100b72edd
